relbench/external/loader.py

```diff
@@ -44,7 +44,7 @@ class SparseTensor:
     def __init__(
         self,
         sparse_tensor: Tensor,
-        device: torch.device | None = None,
+        device: Union[torch.device, None] = None,
```

Comment:
Why? The current one is a recommended type annotation for Python 3.10.

Reply:
I can change back - i am usign Python 3.9 so this was throwing an error. Should we be aware of this kind of back-compatability?

Reply:
Got it, if it throws an error in your env, feel free to change it. I think many people are still using 3.9